sqlglot/parser.py

```diff
@@ -327,6 +330,22 @@ def _parse_insert(self):
             overwrite=overwrite,
         )
 
+    def _parse_update(self):
+        self._match(TokenType.UPDATE)
```

Comment:
you don't need this extra match here, you already do it on line 243

Reply:
Thanks, I didn't notice that.